devenv/src/tasks.rs

```diff
@@ -836,6 +840,21 @@ impl TasksUi {
     }
 }
 
+/// Escape a shell variable by wrapping it in single quotes.
+/// Any single quotes within the variable are escaped.
+fn shell_escape(s: &str) -> String {
```

Comment:
I wonder if we should instead use some library like https://docs.rs/shell-quote/latest/shell_quote/ not to maintain this code which is tricky to get right

Reply:
Yeah, maybe. The above is what `nix develop` does, which felt like a safe choice without getting into the weeds.